engine/table/src/test/java/io/deephaven/engine/table/impl/sort/timsort/BaseTestCharTimSortKernel.java

```diff
@@ -356,11 +393,13 @@ static private void verify(int size, List<CharLongTuple> javaTuples, CharChunk c
             final char timSorted = charChunk.get(ii);
             final char javaSorted = javaTuples.get(ii).getFirstElement();
 
-            final long timIndex = rowKeys.get(ii);
+            final long timIndex = rowKeys == null ? 0 : rowKeys.get(ii);
             final long javaIndex = javaTuples.get(ii).getSecondElement();
 
             TestCase.assertEquals("values[" + ii + "]", javaSorted, timSorted);
-            TestCase.assertEquals("rowKeys[" + ii + "]", javaIndex, timIndex);
+            if (rowKeys != null) {
+                TestCase.assertEquals("rowKeys[" + ii + "]", javaIndex, timIndex);
```

Comment:
Can't `timIndex` and `javaIndex` initialization be moved inside the `if (rowKeys != null)` scope?